tests/__init.test.js

```diff
@@ -5,60 +5,67 @@
  */
 window.debug = true;
 
-navigator.xr = navigator.xr || {};
-navigator.xr.isSessionSupported = function (_sessionType) { return Promise.resolve(true); };
-navigator.xr.requestSession = function (_mode) {
-  const xrSession = new EventTarget();
-  xrSession.supportedFrameRates = [90];
-  xrSession.requestReferenceSpace = function () { return Promise.resolve(); };
-  return Promise.resolve(xrSession);
-};
-
 /* WebXR Stub */
-navigator.xr = navigator.xr || {};
+if (!navigator.xr) {
+  navigator.xr = {};
+}
```

Comment:
Because the tests are now using ESM, this implicitly enabled strict-mode as well, which prevents a lot of overwriting of getter-only or read-only properties. In this case the `xr` property on `navigator` is getter-only when defined.